src/Storage/Device/S3.php

```diff
@@ -365,13 +416,10 @@ public function write(string $path, string $data, string $contentType = ''): boo
      */
     public function move(string $source, string $target): bool
```

Comment:
With this new implementation, could we move the `move` method implementation to the abstract class and avoid from implementing it in each adapter?

Reply:
updated